src/Compilers/Core/Portable/Serialization/ObjectWriter.cs

```diff
@@ -266,6 +266,47 @@ public void WriteValue(object value)
             }
         }
 
+        public void WriteValue(ReadOnlySpan<byte> span)
```

Comment:
Can we test this code? Is there a corresponding deserializer?

Reply:
> Can we test this code?

I'll prepare tests in a separate PR.

> Is there a corresponding deserializer?

It deserializes as `byte[]`.

Reply:
> I'll prepare tests in a separate PR.

Is there some special circumstance that warrants an exception to usual process?

Reply:
The change would not meet the bar for preview 1 approval after the snap, and the time to complete a Roslyn build would push it past that point. The separate PR would allow review of the new tests without causing this PR to get delayed. In the end it's up to y'all, as I see this as more of an opportunistic change than a meaningful one.

Reply:
I don't feel we need to rush. Seems fine if we miss preview 1 for this change.

Reply:
Tests are now added.